lib/game.rb

```diff
@@ -0,0 +1,26 @@
+class Game
+  attr_reader :stat_hash
+
+  def initialize()
+    @stat_hash = {}
+  end
+
+  def create_stat_hash_keys(game_data)
+    index = 0
+    game_data[0].size.times do
```

Comment:
Note to self (Priya) IN HUMAN SPEAK:
I call passed in argument. I see a counter accumulator called index (currently set to 0). This will read element 1, call it's size for .times and DO

- calls empty hash @stat_hash

Uh oh... I need more details.

Logan, can we please talk through this method?

Reply:
Let me explain this to you through a video call